jax/_src/util.py

```diff
@@ -493,6 +492,18 @@ def maybe_named_axis(axis, if_pos, if_named):
     named = True
   return if_named(axis) if named else if_pos(pos)
 
+def get_logger() -> logging.Logger:
```

Comment:
It's not clear this function serves any purpose over
logger = logging.getLogger(__name__)

Just inline it into its users? It would also solve the circular import problem.

Reply:
I think yes to the `getLogger` question.

I think no to the `NullHandler` question. I think the default behavior is the correct one. We do want these messages printed in the absence of any logging configuration (https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library).